Update gem to 0.28.0

bruce-fs opened the pull request:
- Change earnings & impressions to estimated_revenue and ad_impressions
- source:
  https://developers.google.com/youtube/analytics/revision_history#august-10-2016

treble37 committed:
Update gem to 0.26.2

- Change earnings & impressions to estimated_revenue and ad_impressions
- source:
https://developers.google.com/youtube/analytics/revision_history#august-10-2016

claudiofullscreen commented:
@bruce-fs This PR is a breaking change. 

Anyone using yt would have to rename `channel.earnings` into `channel.estimated_revenue` in the code.

Instead, we should just fix the internal implementation (calls to YouTube API) while leaving the external API unchanged.

treble37 committed:
Bump version to 0.28.0

treble37 committed:
Update changelog

- Update changelog w/upgrade instructions
- Change soon to be deprecations (earnings and impressions)

claudiofullscreen commented on CHANGELOG.md:
```diff
@@ -6,6 +6,18 @@ For more information about changelogs, check
+YouTube API as of [November 4, 2016](https://developers.google.com/youtube/analytics/revision_history#august-10-2016).
+
+* [breaking change] Change deprecated `earnings` to `estimated_revenue`
+* [breaking change] Change deprecated `impressions` to `ad_impressions`
```
@bruce-fs  The two lines above are better off like these:

``` md
* [REMOVAL] Remove `#earnings` method for channels, playlists, videos and video groups
* [REMOVAL] Remove `#impressions` method for channels, videos and video groups
* [FEATURE] Add `#estimated_revenue` method for channels, videos and video groups
* [FEATURE] Add `#ad_impressions` method for channels, videos and video groups
```

The deprecation is already explained above.

claudiofullscreen replied:
@bruce-fs Since everything else is fine, I'll merge this PR, but please change the lines above in a separate PR.
